plexus-compiler-its/src/main/it/aspectj-compiler/src/main/java/org/acme/do-not-try-to-compile.txt

```diff
@@ -0,0 +1,7 @@
+The Maven Compiler (MC) API only allows a single source file extension, such as '.java'.
```

Comment:
```suggestion
The Apache Maven Compiler (MC) API only allows a single source file extension, such as '.java'.
```

Reply:
Why is the branding necessary here?

Reply:
as long as you mention Maven it should Apache Maven. That's Apache branding requirement for any Apache product

Reply:
OMG. Fine. This is exactly the kind of "code" review I was afraid of, when we first talked in the issue over atmy AspectJ Maven fork - sorry, AspectJ _Apache_ Maven fork.